gnovm/stdlibs/std/emit_event.go

```diff
@@ -0,0 +1,44 @@
+package std
+
+// ref: https://github.com/gnolang/gno/pull/853
+
+import (
+	"fmt"
+
+	"github.com/gnolang/gno/tm2/pkg/sdk"
+)
+
+type Event struct {
+	Type       string
+	Attributes []sdk.EventAttribute
+}
+
```

Comment:
This event type should also include Realm / Package information (path)

Reply:
hmm.. I had the following thought. However, I'm not sure whether to process the package address by taking a string argument as shown below, or whether I should use `ExecContext.OrigPkgAddr`.

```go
func NewEventWithContext(ctx *ExecContext, typ string, pkgPath string, attrs ...string) (Event, error) {
    var eventAttrs []sdk.EventAttribute
    // ...
    return Event{
        Type:       typ,
        PkgPath:    pkgPath,
        Height:     ctx.Height,
        Timestamp:  ctx.Timestamp,
        Attributes: eventAttrs,
    }, nil
}
```